conditional/blueprints/major_project_submission.py

```diff
@@ -65,7 +70,7 @@ def submit_major_project(user_dict=None):
         return jsonify({"success": False}), 400
     project = MajorProject(user_dict['username'], name, description)
 
-    send_slack_ping({"text":f"<!subteam^S5XENJJAH> {user_dict['username']} submitted their major project, *{name}*!"})
+    send_slack_ping({"text":f"<!subteam^S5XENJJAH> {context_processors.get_member_name(user_dict['username'])} *{user_dict['username']}* submitted their major project, *{name}*!"})
```

Comment:
You shouldn't need to specify `context_processors` if you did what you did on line 1... should be able to just do `get_member_name`

Reply:
Also nit on the formatting, I think it would be easier to read if the username was in parentheses, not boldened. We know people by name first and the username is extra into